package.json

```diff
@@ -25,6 +25,7 @@
     "acorn-jsx-walk": "^2.0.0",
     "acorn-walk": "^8.3.2",
     "arg": "^5.0.2",
+    "babel-plugin-transform-barrels": "github:cmfrydos/babel-plugin-transform-barrels",
```

Comment:
This should be a dev dependency. That should also fix up a lot of unneeded changes in package.lock.

Reply:
Didn't know the meaning of dev packages until I googled it yesterday. But good catch, would probably have forgotten to fix it.